src/Server/Event/DoctrineResourceEvent.php

```diff
@@ -48,18 +48,12 @@ class DoctrineResourceEvent extends Event
     /** @var ObjectManager */
     protected $objectManager;
 
-    /**
-     * @return ObjectManager
-     */
-    public function getObjectManager()
+    public function getObjectManager(): ObjectManager
```

Comment:
add return type for public method is BC break for non-final class, ref https://3v4l.org/egUAq